dev-haskell/gargoyle-postgresql/gargoyle-postgresql-0.2.0.1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Manage PostgreSQL servers with gargoyle"
+HOMEPAGE="https://hackage.haskell.org/package/gargoyle-postgresql"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="enable-psql-test executables"
+
+PATCHES=( "${FILESDIR}/${P}-add-executables-flag.patch" )
+
+RDEPEND=">=dev-haskell/gargoyle-0.1.1.0:=[profile?] <dev-haskell/gargoyle-0.2:=[profile?]
+	>=dev-haskell/posix-escape-0.1:=[profile?] <dev-haskell/posix-escape-0.2:=[profile?]
+	>=dev-haskell/stringsearch-0.3:=[profile?] <dev-haskell/stringsearch-0.4:=[profile?]
+	>=dev-haskell/text-1.2:=[profile?] <dev-haskell/text-1.3:=[profile?]
+	>=dev-lang/ghc-8.6.3:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.4.0.1
+"
+
+src_configure() {
+	haskell-cabal_src_configure \
+		$(cabal_flag enable-psql-test)\
```

Comment:
I'm just going to go ahead and remove this USE flag, and leave the `test` USE flag alone.

Reply:
Ah, I see that there are no other test-suites defined in the cabal file. I will take another look.

Reply:
Since these tests most likely spin up an instance of postgresql, I would think they are not appropriate for gentoo's sandbox. I'm going to remove the USE flag, remove `test-suite` from `CABAL_FEATURES`, and add `RESTRICT=test`.